<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SectionViewDock</class>
 <widget class="QDockWidget" name="SectionViewDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>680</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>971</width>
    <height>411</height>
   </size>
  </property>
  <property name="floating">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Section View</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QVTKWidget" name="vtkView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Field:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="fieldBox"/>
      </item>
      <item>
       <widget class="QToolButton" name="exportImageButton">
        <property name="toolTip">
         <string>export image</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../resources/resources.qrc">
          <normaloff>:/icons/picture_save.png</normaloff>:/icons/picture_save.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="pqCollapsedGroup" name="collapsedGroup">
      <property name="title">
       <string>Section Parameters</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Source</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QComboBox" name="selModeBox">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="text">
               <string>Section Thickness</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="thicknessBox">
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="applyButton">
              <property name="text">
               <string>Apply</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Start</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="startXBox">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="startYBox">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="startZBox">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="startPointButton">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>End</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="endXBox">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="endYBox">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="endZBox">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="endPointButton">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="geoDataGroup">
         <property name="minimumSize">
          <size>
           <width>240</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>240</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Geo Data</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqCollapsedGroup</class>
   <extends>QGroupBox</extends>
   <header>pqCollapsedGroup.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
